synthtool/gcp/templates/python_samples/README.md

```diff
@@ -37,15 +37,14 @@ To run this sample:
 {% if metadata['repo']|length %}{% if metadata['repo']['client_library'] %}
 These samples use the [Google Cloud Client Library for Python][client_library_python].{% endif %}
 You can read the documentation for more details on API usage and use GitHub
-to [browse the source][source] and [report issues][issues].{% endif %}
+to <a href="https://github.com/{{ metadata['repo']['repo'] }}">browse the source</a> and [report issues][issues].{% endif %}
 
 ### Contributing
-For [contributing guidelines][contrib_guide], the [Python style guide][py_style], and more information on prerequisite steps to contribute, view the source code at <a href="https://github.com/{{ metadata['repo']['repo'] }}">{{ metadata['repo']['repo'] }}</a>.
+View the [contributing guidelines][contrib_guide], the [Python style guide][py_style]. More information on prerequisite steps to contribute may be available at the source code.
```

Comment:
I would just make this read, "View the contributing guidelines and the Python style guide for more information."